extensions/vscode/lsp-server/wasm/lsp-wake.pre.js

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2022 SiFive, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You should have received a copy of LICENSE.Apache2 along with
+ * this software. If not, you may obtain a copy at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// <reference types="emscripten" />
+
+var Module = {
+    'locateFile': function (wasmBinaryFile, prefix) {
+        if (ENVIRONMENT_IS_NODE) {
+            return prefix + "/../wasm/" + wasmBinaryFile; // TODO: understand why a hash file is needed
```

Comment:
Just noting that we still haven't solved this. I'm happy to come back to this later though, no need to change it here. It looks like with `ENVIRONMENT_IS_NODE` you made it work in both situations and everything is still using relative paths which is good.